test/src/models/form_group_test.dart

```diff
@@ -137,7 +137,7 @@ void main() {
     test('Get value returns a Map with controls name and values', () {
       final form = FormGroup({
         'name': FormControl(value: 'john'),
-        'email': FormControl(value: 'john@email.com'),
+        'email': FormControl<String>(value: 'john@email.com'),
```

Comment:
This will not allow creating a control with dynamic type as:
```dart
final control = FormControl(
   value: "I'm a String control",
   validators: [Validators.email], // exception is thrown
);
```
So I suggest leaving the Email validator as **dynamic**